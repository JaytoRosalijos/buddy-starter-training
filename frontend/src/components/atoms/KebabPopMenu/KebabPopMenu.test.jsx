import renderer from 'react-test-renderer';
import { KebabPopMenu } from '.';

describe("[Atom] Kebab Pop Menu", () => {

    it("Kebab Pop Menu renders correctly without props", async () => {
        const tree = renderer.create(<KebabPopMenu />).toJSON();
        expect(tree).toMatchSnapshot();
    });

    it("Kebab Pop Menu renders correctly with props", async () => {
        const tree = renderer.create(<KebabPopMenu 
                                        isActiveKebab
                                        onUpdate={() => {}} 
                                        onDelete={() => {}}
                                        onOpenMenu={() => {}}
                                        onCloseMenu={() => {}}
                                    />).toJSON();
        expect(tree).toMatchSnapshot();
    });

});
