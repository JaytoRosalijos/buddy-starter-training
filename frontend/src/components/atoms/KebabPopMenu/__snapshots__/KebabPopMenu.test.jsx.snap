// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Atom] Kebab Pop Menu Kebab Pop Menu renders correctly with props 1`] = `
<svg
  aria-describedby="popup-2"
  className="sc-hLBbgP icHdEJ"
  onClick={[Function]}
>
  kebab.svg
</svg>
`;

exports[`[Atom] Kebab Pop Menu Kebab Pop Menu renders correctly without props 1`] = `
<svg
  aria-describedby="popup-1"
  className="sc-hLBbgP cGrljD"
  onClick={[Function]}
>
  kebab.svg
</svg>
`;
